homeassistant/helpers/selector.py

```diff
@@ -1198,6 +1199,17 @@ def __init__(self, config: ObjectSelectorConfig | None = None) -> None:
         """Instantiate a selector."""
         super().__init__(config)
 
+    def serialize(self) -> dict[str, dict[str, ObjectSelectorConfig]]:
+        """Serialize ObjectSelector for voluptuous_serialize."""
+        _config = deepcopy(self.config)
+        if "fields" in _config:
+            for items in _config["fields"].values():
+                if isinstance(items["selector"], Selector):
+                    items["selector"] = {
+                        items["selector"].selector_type: items["selector"].config
```

Comment:
`items` could be renamed to `field` for additional clarity:
```suggestion
            for field in _config["fields"].values():
                if isinstance(field["selector"], Selector):
                    field["selector"] = {
                        field["selector"].selector_type: field["selector"].config
```

Reply:
Don't see why that would bring additional clarity.